services/DynamoService.ts

```diff
@@ -1,37 +1,29 @@
 import { AWSError, DynamoDB } from 'aws-sdk';
 import { ScanInput, ScanOutput, QueryOutput } from 'aws-sdk/clients/dynamodb';
-import { DynamoConfig, MonitoredService, ServiceHealthCheckResponse } from '../types';
-import logger from '../config/logger';
 import { AttributeMap } from 'aws-sdk/clients/dynamodbstreams';
+import { DynamoConfig, MonitoredService, ServiceHealthCheckResponse } from '../types';
 import { DevPortalError } from '../models/DevPortalError';
 
 export type FilterParams = Pick<ScanInput, 'ExpressionAttributeValues' | 'FilterExpression'>;
 
 export default class DynamoService implements MonitoredService {
   public client: DynamoDB.DocumentClient;
+
   public dynamo: DynamoDB;
 
-  constructor(config: DynamoConfig) {
+  public constructor(config: DynamoConfig) {
     this.dynamo = new DynamoDB(config);
     this.client = new DynamoDB.DocumentClient(config);
   }
 
   public putItem(item: Record<string, unknown>, tableName: string): Promise<void> {
-    // The DynamoDB API breaks if empty strings are passed in
```

Comment:
Is this not true anymore?

Reply:
By marshalling the item before passing it on it removes empty strings.
So it's technically true that if you were to pass an empty string to `putItem` before marshalling it, yes it would still break. Christine and I discussed moving the marshall to this function for that reason but it would've been a bit more work and this PR was supposed to be about eslint 😝